ui/public/static/style/index.css

```diff
@@ -0,0 +1,6 @@
+.error-autocomplete-value .MuiOutlinedInput-notchedOutline {
```

Comment:
Why custom css?

Reply:
when input error , the border will be red to remind people should input a current value